lib/osml/osml_vpc.ts

```diff
@@ -110,6 +112,7 @@ export class OSMLVpc extends Construct {
       // Create a new VPC
       const vpc = new Vpc(this, "OSMLVPC", {
         vpcName: props.vpcName,
+        maxAzs: isIsoB ? 2 : 3,
```

Comment:
I feel like maxAzs should be moved into the configuration of the construct and then this should be set to 2 or 3 when it is created via the config.

Reply:
I agreed. But its only pertaining to one region which only does 2. Majority of it have 3.